chorus_book/src/guide-projector.md

```diff
@@ -4,19 +4,22 @@ Projector is responsible for performing the end-point projection and executing t
 
 ## Creating a Projector
 
-To create a `Projector`, you need to provide the location and the transport.
+To create a `Projector`, you need to provide the set of locations it can work with, the target location, and the transport.
 
 ```rust
 # extern crate chorus_lib;
 # use chorus_lib::transport::local::LocalTransport;
 # use chorus_lib::core::{ChoreographyLocation, Projector};
+# use chorus_lib::{hlist, projector};
 # let transport = LocalTransport::from(&[Alice::name(), Bob::name()]);
 # #[derive(ChoreographyLocation)]
 # struct Alice;
 # #[derive(ChoreographyLocation)]
 # struct Bob;
 #
-let projector = Projector::new(Alice, transport);
+
+type AL = hlist!(Alice, Bob);
```

Comment:
I don't want users to think that they have to declare location sets as types. I prefer just using hlist! inside projector! like `projector!(hlist!(Alice, Bob), Alice, transport)`